config/config.exs

```diff
@@ -29,8 +28,10 @@ use Mix.Config
 #
 #     import_config "#{Mix.env()}.exs"
 
-config :logger, :console, format: "$message\n", colors: [enabled: false]
-
 config :event_logger, :environment, Mix.env()
 
+config :event_logger, time_api: EventLogger.Time.DateTime
```

Comment:
Is this format UTC?

Reply:
Yes sam if you look at the function here you can see that it just returns the results of DateTime.utc_now

https://github.com/JoeARO/event_logger/blob/master/lib/event_logger/time/date_time.ex#L6

This was implemented so the time could be mocked for testing

Reply:
I assumed it was. Just checking in case we needed to set GMT or specify a UK timezone somewhere.